pilot/pkg/model/config_test.go

```diff
@@ -29,6 +29,7 @@ import (
 	mccpb "istio.io/api/mixer/v1/config/client"
 	networking "istio.io/api/networking/v1alpha3"
 	rbacproto "istio.io/api/rbac/v1alpha1"
+	authz "istio.io/api/security/v1beta1"
```

Comment:
NIT: Should we call it "security" instead of "authz"? Same below.

Reply:
I think it's okay either way, the package only includes authorization, we can change it when we later adding authentication